docs/content/preview/architecture/transactions/read-committed.md

```diff
@@ -27,24 +27,24 @@ The following two key semantics set apart Read Committed isolation from Repeatab
+* A client starts a distributed transaction by connecting to YSQL on a node `N1` in the YugabyteDB cluster and issues a statement which reads data from multiple shards on different physical YB-TServers in the cluster. For this issued statement, the read point which defines the snapshot of the database at which the data will be read, is picked on a YB-TServer node `M` based on the current time of that YB-TServer. Depending on the scenario, node `M` could be the same as `N1` or not, but that is not relevant to this discussion. Consider `T1` to be the chosen read time.
 * The node `N1` might collect data from many shards on different physical YB-TServers. In this pursuit, it will issue requests to many other nodes to read data.
-* Assuming that node `N1` reads from node `N2`, it could be the case that there exists some data written on node `N2` at time `T2` (> `T1`) but had been written before the read was issued. This can happen because of clock skew where the physical clock on node `N2` might be running slightly ahead of node `M`, and hence the write which was actually done in the past, still has a timestamp higher than `T1`.
+* Assuming that node `N1` reads from node `N2`, it could be the case that there exists some data written on node `N2` with a write timestamp `T2` (> `T1`) but it had been written before the read was issued. This can happen because of clock skew where the physical clock on node `N2` might be running ahead of node `M`, and hence the write which was actually done in the past, still has a write timestamp higher than `T1`.
 
-  Note that the clock skew between all nodes in the cluster is always in a [`max_clock_skew`](/preview/reference/configuration/yb-tserver/#max-clock-skew-usec) bound due to clock synchronization algorithms.
-* For writes at some time later than `T1` + `max_clock_skew`, the database can be sure that they were done after the read timestamp was chosen on any node. But for writes at a time between `T1` and `T1` + `max_clock_skew`, node `N2` can find itself in an ambiguous situation such as the following:
+  Note that the clock skew between all nodes in the cluster is always in a [max_clock_skew_usec](/preview/reference/configuration/yb-tserver/#max-clock-skew-usec) bound due to clock synchronization algorithms.
+* For writes with a write timestamp higher than `T1` + `max_clock_skew`, the database can be sure that they were done after the read timestamp was chosen. But for writes at a time between `T1` and `T1` + `max_clock_skew`, node `N2` can find itself in an ambiguous situation such as the following:
```

Comment:
Suggestion:

* For writes with a write timestamp later than `T1` + `max_clock_skew`, the database can be sure that these writes were done after the read timestamp had been chosen. But for writes at a time between `T1` and `T1` + `max_clock_skew`, node `N2` can find itself in an ambiguous situation, such as the following:

Reply:
incorporated the suggestion with this minor deviation: I used `.... for writes with a write timestamp between .... ` instead of `... for writes at a time between ... ` since the former is more accurate. Because we are specifically talking about a `write timestamp` here